app/assets/javascripts/application.js

```diff
@@ -20,6 +20,7 @@
 //= require jquery-file-upload/js/jquery.iframe-transport.js
 //= require flot/jquery.flot.js
 //= require flot/jquery.flot.crosshair.js
+//= require frappe-charts.min.iife
```

Comment:
Lets remove this line here and use the cdn link : 
<script src="https://cdn.jsdelivr.net/npm/frappe-charts@0.0.8/dist/frappe-charts.min.iife.js"></script>
in **application.html.erb** just above the end of head tag .

Reply:
Okk I would do that

Reply:
@sagarpreet-chadha It worked out. Thanks

Reply:
Awesome !